addons/sale_timesheet/views/project_task_views.xml

```diff
@@ -145,6 +145,20 @@
             </field>
         </record>
 
+        <record id="view_task_form2_inherited1" model="ir.ui.view">
+            <field name="name">project.task.form.inherited</field>
+            <field name="model">project.task</field>
+            <field name="inherit_id" ref="hr_timesheet.view_task_form2_inherited" />
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='child_ids']/tree/field[@name='remaining_hours']" position="after">
+                    <field name="remaining_hours_so" string="Remaining Hours on SO" widget="timesheet_uom" optional="hide"/>
+                </xpath>
+                <xpath expr="//field[@name='depend_on_ids']/tree/field[@name='remaining_hours']" position="after">
+                    <field name="remaining_hours_so" string="Remaining Hours on SO" widget="timesheet_uom" optional="hide"/>
+                </xpath>
```

Comment:
Why not putting this in `view_sale_service_inherit_form2` form view in the same module? 🤔

Reply:
it was added after my PR and i got conflicts in this file, i will merge them